scripts/wrappers/start.sh

```diff
@@ -3,11 +3,10 @@
 set -eu
 
 
-# source "${OPS_ROOT}"/helpers/snap-logger.sh "daemon"
+source "${OPS_ROOT}"/helpers/snap-logger.sh "daemon"
```

Comment:
I think we can comment this back, it's mainly useful for debugging the snap build